datacapturing/src/main/java/de/cyface/datacapturing/backend/CapturingProcess.java

```diff
@@ -47,6 +47,10 @@
      * A delay used to bundle capturing of sensor events, to reduce power consumption.
      */
     private static final int SENSOR_VALUE_DELAY_IN_MICROSECONDS = 500_000;
+    /**
+     * A delay used to reduce capturing of sensor events, to reduce data size. 10 k = 100 Hz
+     */
+    private static final int DELAY_BETWEEN_SENSOR_EVENTS_IN_MICROSECONDS = 10_000;
```

Comment:
This should probably be a setting one can provide to the DataCapturingService. Ideally it should be in Hertz, like it is in iOS. Should suffice to make a task from this comment.

Reply:
Okay: https://cyface.atlassian.net/browse/MOV-688